wp-multi-network/includes/classes/class-wp-ms-networks-api.php

```diff
@@ -0,0 +1,859 @@
+					'type'         => 'string',
+					'context'      => array( 'view', 'edit', 'embed' ),
+				),
+				'path'           => array(
+					'description'  => __( 'The path of the network object', 'wp-multi-network' ),
+					'type'         => 'string',
+					'context'      => array( 'view', 'edit', 'embed' ),
+				),
+				'link'             => array(
+					'description'  => __( 'URL to the object.', 'wp-multi-network' ),
+					'type'         => 'string',
+					'format'       => 'uri',
+					'context'      => array( 'view', 'edit', 'embed' ),
+					'readonly'     => true,
+				),
+			),
+		);
+
+		return $this->add_additional_fields_schema( $schema );
+	}
+
+	/**
+	 * Retrieves the query params for collections.
+	 *
+	 * @since 2.2.0
+	 *
+	 * @return array Networks collection parameters.
+	 */
+	public function get_collection_params() {
+		$query_params = parent::get_collection_params();
+
+		$query_params['context']['default'] = 'view';
+
+		$query_params['domain'] = array(
+			'description'       => __( 'Limit result set to networks assigned to specific domains.', 'wp-multi-network' ),
+			'type'              => 'array',
+			'items'             => array(
+				'type'          => 'string',
+			),
+		);
+
+		$query_params['domain_exclude'] = array(
+			'description'       => __( 'Ensure result set excludes networks assigned to specific domain.', 'wp-multi-network' ),
+			'type'              => 'array',
+			'items'             => array(
+				'type'          => 'string',
+			),
+		);
+
+		$query_params['exclude'] = array(
+			'description'        => __( 'Ensure result set excludes specific IDs.', 'wp-multi-network' ),
+			'type'               => 'array',
+			'items'              => array(
+				'type'           => 'integer',
+			),
+			'default'            => array(),
+		);
+
+		$query_params['include'] = array(
+			'description'        => __( 'Limit result set to specific IDs.', 'wp-multi-network' ),
+			'type'               => 'array',
+			'items'              => array(
+				'type'           => 'integer',
+			),
+			'default'            => array(),
+		);
+
+		$query_params['offset'] = array(
+			'description'        => __( 'Offset the result set by a specific number of items.', 'wp-multi-network' ),
+			'type'               => 'integer',
+		);
+
+		$query_params['order']      = array(
+			'description'           => __( 'Order sort attribute ascending or descending.', 'wp-multi-network' ),
+			'type'                  => 'string',
+			'default'               => 'desc',
+			'enum'                  => array(
+				'asc',
+				'desc',
+			),
+		);
+
+		$query_params['orderby']    = array(
+			'description'           => __( 'Sort collection by object attribute.', 'wp-multi-network' ),
+			'type'                  => 'string',
+			'default'               => 'id',
+			'enum'                  => array(
+				'id',
+				'domain',
+				'path',
+				'domain_length',
+				'path_length',
+				'network__in',
```

Comment:
This should be `include` instead of `network__in`, like the parameter itself. Then, it needs to be dynamically adjusted if used.